apps/native-component-list/ts-declarations/env.d.ts

```diff
@@ -0,0 +1,9 @@
+declare global {
+  namespace NodeJS {
+    interface ProcessEnv {
+      EXPO_PUBLIC_TEST: string;
+    }
+  }
+}
+
+export {};
```

Comment:
If we want to leave it, can you add some comments here to indicate that it was added for a test purposes?